test/manual/amp-video.amp.html

```diff
@@ -223,6 +223,14 @@ <h2>Analytics Events</h2>
             "vars": {
               "eventType": "video-ended"
             }
+          },
+          "myVideoCustom": {
+            "on": "video-custom",
```

Comment:
Should this be `video-hosted-custom`?

Reply:
On the contrary, `video-hosted-custom` is the name that can't be used.

Reply:
Does this implementation allow document authors to create their own event names? I think this will be more dangerous than namespacing the custom events under a top level "customEvent" name, because it opens the possibility that publishers will create their own `"on": "blah"` events that may conflict with first-party event we make in the future.

Reply:
We could force a prefix like `video-custom` to avoid naming collisions.

Reply:
Sounds good, that should set a good expectation so that authors don't go wild and try to make their own `"on": "video-played"` events that block ours or something.